src/chexus/__main__.py

```diff
@@ -20,9 +20,28 @@ def main():
     parser.add_argument(
         '--checksums', action='store_true', help='Compute and print checksums'
     )
+    parser.add_argument(
+        '--no-scipp', action='store_true', help='Skip the validators that require scipp'
```

Comment:
You missed the obvious pun of using `--skip-scipp`.

Just kidding, please don't do this ;)

Reply:
I was sorely tempted...